src/types/Endpoints.ts

```diff
@@ -352,4 +352,44 @@ export interface IEditPlaylistRequest extends ObjectSnakeToCamel<EditPlaylistEnd
     playlistDescription?: string;
     playlistName?: string;
   }[];
+}
+
+export interface IReelWatchRequest {
+  context: {},
```

Comment:
You don't need the context out here (see message above about why).

Reply:
Removed